src/libraries/System.Console/src/System/ConsolePal.Unix.cs

```diff
@@ -722,6 +692,27 @@ private static Encoding GetConsoleEncoding()
                 Encoding.Default;
         }
 
+#pragma warning disable IDE0060
```

Comment:
Another set of examples for why IDE0060 should probably instead be updated to not warn if the method always throws.